app/src/main/java/org/p2p/wallet/common/ui/BaseFragmentAdapter.kt

```diff
@@ -13,10 +13,10 @@ class BaseFragmentAdapter(
     private val fragmentManager: FragmentManager,
     lifecycle: Lifecycle,
     private val items: List<KClass<out Fragment>>,
-    var args: List<Bundle>? = null
+    var args: List<Bundle?>? = null
```

Comment:
do we really need to make Bundle nullable instead of just empty list?

Reply:
it is because it is related to 2 lists
first one is list of classes second one is list on args

I think we can refactor it and make object which contains class and bundle if needed and it will looks more clear

what do you think ?